learn/getting-started/getting-started-with-ballerina.md

```diff
@@ -0,0 +1,162 @@
+```
+layout: getting-started-with-ballerina-left-nav-pages
+title: Getting Started with Ballerina
+description: Let’s get you set up a Ballerina development environment and write a simple Ballerina program.
+keywords: ballerina, programming language, ballerina packages, getting started
+permalink: /learn/getting-started-with-ballerina
+active: getting-started-with-ballerina
+intro: Let’s get you set up a Ballerina development environment and write a simple Ballerina program.
+---
+```
+
+Let’s get you set up a Ballerina development environment and write a simple Ballerina program.
+
+## Prerequisites
+
+To complete this tutorial, you need:
+
+- A command terminalA text editor.
+- [VSCode](https://code.visualstudio.com/) with the [Ballerina extension](https://marketplace.visualstudio.com/items?itemName=WSO2.ballerina) installed is our preferred choice, and it has good support for Ballerina.
+- A Ballerina installation. Follow the steps in [Installing Ballerina](https://ballerina.io/learn/installing-ballerina/setting-up-ballerina/).
+- Some experience with at least one programming language. 
+
+## Meet `bal`
+
+`bal` is the Ballerina build tool and package manager. Among other things, `bal` helps you to create, build, test, and run your project. 
+
+You can get the latest `bal` tool version with the Ballerina installation. Open your terminal and run the following commands just to make sure everything is ready:
+
+```
+$bal
+$bal version
+```
+
+## Create a new project
+
+Let’s write a Ballerina program that prints “Hello, World!”. Use the `bal new` command to create a new Ballerina project. 
```

Comment:
```suggestion
Let’s write a Ballerina program that prints “Hello, World!”. Execute the `bal new` command to create a new Ballerina project. 
```

Reply:
I see nothing wrong with `use` here. IMO, it is better than execute in this context